src/OmniSharp.Abstractions/Services/IUpdates.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace OmniSharp.Services
+{
+    public interface IUpdates
```

Comment:
Why not derive an IWaitableProjectSystem from IProjectSystem and have MSBuild and Dotnet implement that?

Reply:
Done